### DISCUSSION TITLE
Why does an additional window pop up when I execute the EXE after packaging it?

### DISCUSSION THREAD

**Discussion by daydaystudylab** [slint-ui/slint#7073] [Q&A]
Why does an additional window pop up when I execute the EXE after packaging it?

**Comment by daydaystudylab** [comment]
[![dfgd](https://github.com/user-attachments/assets/50eeadda-0941-4190-b52c-d12f25068111)
](url)

**Comment by ffred** [comment]
Hi,
not sure if I understand exactly what you mean but maybe you just need to add this line at the very beginning of your main.rs file to disable the command window :
`#![windows_subsystem = "windows"]`

**Comment by hunger** [comment]
I think this is the issue behind what you are seeing: https://github.com/slint-ui/slint/issues/3235

We are working on new docs that actually mention this:-)
